### ISSUE TITLE
require alt text description is vague as to what it actually does

### ISSUE THREAD

**Issue by DurinMephit** [bluesky-social/social-app#1252]
**Describe the bug**
Under Accessibility in the app, there is a toggle with the following description:  "Require alt text on images."  This is potentially confusing.  To varying individuals, it could mean:
- Require alt text on images _when posting_ - i.e., force me to fill out the alt text field
- Require alt text on images _in my feed_ - i.e., as a visually impaired individual, hide content that doesn't have alt text because it's not useful to me

**To Reproduce**

1. Open the hamburger menu
2. Open Settings
3. Scroll down to or look under Accessibility
4. Contemplate the meaning of the words "Require alt text on images"

**Expected behavior**

The label should be clearer about the context in which it applies.  Is it when posting?  Is it when viewing?

**Screenshots**
![Untitled](https://github.com/bluesky-social/social-app/assets/70461208/5bb37047-dc40-4031-9483-523588dd3b03)

**Details**

 - Platform:  iOS, Web
 - Platform version: 1.46.0
 - App version: 1.46

**Additional context**

Proof that I'm not the only one who was confused by this:  https://bsky.app/profile/durinmephit.bsky.social/post/3k5kob5rzds27

**Comment by simonetthomas** [comment]
I too thought it was to only see pictures that have an alt... I had to see a post of someone talking about it to understand that was supposed to force *me* to add an alt to post.

**Comment by mschwendener** [comment]
And could it be more obvious if the toggle is on "on" or "off"? 

<img width="255" alt="screenshot_4492" src="https://github.com/bluesky-social/social-app/assets/4497705/d1db14f7-15f8-4140-bfec-0c0521d26b75">

([example](https://lh3.googleusercontent.com/22LIcVcqHMXDkIBXurOMcpZMmEyrN3aZAfAM6Z-B1IAvZu_7yMckjAjHz_tFN3PIoJ7w7mnEhiHq_nUr32pKGgMZHscSUGEIGm8exQ=w1064-v0))

**Comment by softyoda** [comment]
I saw this option and deactivated directly it because I wasn't deficient or disabled (I legitimately thought it was about not displaying images without alt in my feed).

But if it's just for enabling a little popup-reminder to add alt on my images, I really think it could be a good idea.

Maybe without changing much of the options, having a popover description text when hoovering the option could help us better understand the context applicable to this option.

**Comment by pfrazee** [comment]
Yeah we should fix this copy, thanks
